mono/metadata/debug-helpers.c

```diff
@@ -1079,14 +1079,19 @@ mono_value_describe_fields (MonoClass* klass, const char* addr)
 void
 mono_class_describe_statics (MonoClass* klass)
 {
+	MonoError error;
 	MonoClassField *field;
 	MonoClass *p;
 	const char *field_ptr;
-	MonoVTable *vtable = mono_class_vtable_full (mono_domain_get (), klass, FALSE);
+	MonoVTable *vtable = mono_class_vtable_full (mono_domain_get (), klass, &error);
 	const char *addr;
 
-	if (!vtable)
+	if (!vtable) {
+		mono_error_cleanup (&error);
 		return;
+	}
+	mono_error_assert_ok (&error);
+
```

Comment:
same as above: is there a case where the error is set, but it doesn't return NULL